### DISCUSSION TITLE
Individual sides inoperative

### DISCUSSION THREAD

**Discussion by 18259669383** [tailwindlabs/tailwindcss#10057] [Help] [answered]
versions：3.2.4
`
      <div class="w-full p-2 text-xl border-b-4 border-indigo-500">
`

**Comment by wongjn** [comment, marked as answer]
Could you give more details about what problem you are experiencing please?

**Reply by 18259669383** [reply]
Did not take effect when placing the bottom border on the block

**Reply by wongjn** [reply]
Thank you for that additional information. It does work in [a Tailwind Play](https://play.tailwindcss.com/HkBfpfyH3A) which suggests that it could be some misconfiguration or something else within your project. You could try checking:
- Tailwind has run a CSS compilation since you added the class.
- The file that the code appears in is covered by the `content` file globs in the Tailwind configuration.
- You are not using string interpolation/concatenation to build the class string.

**Reply by 18259669383** [reply]
No configuration changes.

**Reply by wongjn** [reply]
No configuration changes relative to the Tailwind defaults? Then you should consider adding an entry to the `content` file globs in the Tailwind configuration such that Tailwind scans the file that the code appears in.
